packages/@vuepress/bundler-vite/src/plugin/createPlugin.ts

```diff
@@ -16,6 +17,7 @@ export const createPlugin = ({
   isServer: boolean
   isBuild: boolean
 }): Plugin[] => [
+  createConstantReplacementPlugin(),
   createVuePlugin(options.vuePluginOptions),
   createWorkaroundPlugin(),
```

Comment:
Could you help to switch the position of `createVuePlugin` and `createWorkaroundPlugin` :rofl: